pkg/test-infra/fixture/fixture.go

```diff
@@ -174,6 +175,7 @@ func init() {
 	flag.StringVar(&Context.TiDBConfigFile, "tidb-config", "", "path of tidb config file (cluster A in abtest case)")
 	flag.StringVar(&Context.TiKVConfigFile, "tikv-config", "", "path of tikv config file (cluster A in abtest case)")
 	flag.StringVar(&Context.PDConfigFile, "pd-config", "", "path of pd config file (cluster A in abtest case)")
+	flag.IntVar(&Context.TiKVReplicas, "tikv-replicas", 3, "number of tikv replicas")
```

Comment:
Do we also want to configure this parameter in abtest? If that, maybe we also need to add another parameter for the second cluster.